addons/web/static/src/views/fields/properties/properties_field.js

```diff
@@ -236,10 +236,11 @@ export class PropertiesField extends Component {
     async onPropertyEdit(event, propertyName) {
         event.stopPropagation();
         event.preventDefault();
-        if (!await this.checkDefinitionWriteAccess()) {
-            this.notification.add(_lt("You need to be able to edit parent first to configure property fields"), {
-                type: "warning",
-            });
+        if (!(await this.checkDefinitionWriteAccess())) {
```

Comment:
Prettier = :hankey:

Reply:
I don't agree, it make the look super consistent :)

Reply:
You have the good :socks: and bad :socks: . Prettier is bad :socks: . I will send you @awa-odoo with its :boxing_glove: !